src/main/scala/ch/wavein/sbt/ScalajsTypescriptTypesExportPlugin.scala

```diff
@@ -37,11 +36,10 @@ object ScalajsTypescriptTypesExportPlugin extends AutoPlugin {
     (crossTarget in fastOptJS) := outputDir.value,
     jsOutputName := name.value,
     generateTypescript := {
-      PlatformTokenizerCache.megaCache.clear()
       val outputFile = outputDir.value / (jsOutputName.value + ".d.ts")
       val sources:Seq[Source] = (unmanagedSources in Compile).value.map { file =>
-        // Workaround for https://github.com/scalameta/scalameta/issues/874
-        new ScalametaParser(Input.File(file), dialects.ParadiseTypelevel212).parseSource()
+        // see: https://github.com/scalameta/scalameta/pull/2044
+        new ScalametaParser(Input.File(file), dialects.Scala).parseSource()
```

Comment:
I understand ParadiseTypelevel212 for scalameta macro.

> Now, the Dialect class no longer uses the @data annotation and
> instead this commit manually implements the macro expansion. 

https://github.com/scalameta/scalameta/pull/2044